File: Examples/Output/io_input_output_test.java
import java.util.Scanner;

class Calculadora {
    private static Scanner scanner = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Ingresa el primer numero:");
        int a = Integer.parseInt(scanner.nextLine());
        System.out.println("Ingresa el segundo numero:");
        int b = Integer.parseInt(scanner.nextLine());
        int suma = (a + b);
        int resta = (a - b);
        System.out.println("Suma:");
        System.out.println(suma);
        System.out.println("Resta:");
        System.out.println(resta);
    }
}

File: Examples/Input/io_input_output_test.cs
using System;

class Calculadora
{
    public static void Main(string[] args)
    {
        Console.WriteLine("Ingresa el primer numero:");
        int a = int.Parse(Console.ReadLine());

        Console.WriteLine("Ingresa el segundo numero:");
        int b = int.Parse(Console.ReadLine());

        int suma = a + b;
        int resta = a - b;

        Console.WriteLine("Suma:");
        Console.WriteLine(suma);
        Console.WriteLine("Resta:");
        Console.WriteLine(resta);
    }
}
